src/main/java/io/kestra/plugin/confluence/pages/CreateConfluencePage.java

```diff
@@ -0,0 +1,244 @@
+        String rParentId = runContext.render(this.parentId).as(String.class).orElse(null);
+        String rSubtype = runContext.render(this.subtype).as(String.class).orElse(null);
+        String rStatus = runContext.render(this.status).as(String.class).orElse(null);
+        Boolean rEmbedded = runContext.render(this.embedded).as(Boolean.class).orElse(false);
+        Boolean rMakePrivate = runContext.render(this.makePrivate).as(Boolean.class).orElse(false);
+        Boolean rRootLevel = runContext.render(this.rootLevel).as(Boolean.class).orElse(false);
+
+        // Convert Markdown to HTML
+        MutableDataSet options = new MutableDataSet();
+        Parser parser = Parser.builder(options).build();
+        HtmlRenderer renderer = HtmlRenderer.builder(options).build();
+        Node document = parser.parse(rMarkdown);
+        String htmlBody = renderer.render(document);
+
+        JsonNodeFactory jnf = JsonNodeFactory.instance;
+        ObjectNode payload = jnf.objectNode();
+
+        // Build JSON payload
+        payload.put("spaceId", rSpaceId);
+        payload.put("status", rStatus);
+        payload.put("title", rTitle);
+        payload.put("parentId", rParentId);
+        payload.put("subtype", rSubtype);
+
+        ObjectNode body = payload.putObject("body");
+        body.put("representation", "storage");
+        body.put("value", htmlBody);
+
+        Map<String, Object> queryParams = new LinkedHashMap<>();
+        queryParams.put("embedded", rEmbedded);
+        queryParams.put("private", rMakePrivate);
+        queryParams.put("root-level", rRootLevel);
+        
+
+        String base = rServerUrl.endsWith("/") ? rServerUrl.substring(0, rServerUrl.length() - 1) : rServerUrl;
+        String url = base + "/wiki/api/v2/pages";
+
+        HttpResponse<JsonNode> response = Unirest.post(url)
+            .queryString(queryParams)
+            .basicAuth(rUsername, rApiToken)
+            .header("Accept", "application/json")
+            .header("Content-Type", "application/json")
+            .body(payload)
+            .asJson();
+
+        if (response.getStatus() < 200 || response.getStatus() >= 300) {
+            logger.error("Confluence create failed: status={} body={}", response.getStatus(), response.getBody());
+            throw new IllegalStateException("Confluence API returned " + response.getStatus());
+        }
+
+        String responseBody = response.getBody() != null ? response.getBody().toString() : "";
+        return Output.builder()
+            .child(OutputChild.builder().value(responseBody).build())
+            .build();
+    }
+
+    @Builder
+    @Getter
+    public static class Output implements io.kestra.core.models.tasks.Output {
+        @Schema(
+            title = "The API response from Confluence.",
+            description = "Contains the full JSON response from the Confluence API after creating the page."
+        )
+        private final OutputChild child;
```

Comment:
`OutputChild` is useless encapsulation here I think, just put the `value` here.
Same idea for other Output in other tasks.